### PR TITLE
[UT]Add test WithNodeNum in nutsdb/optins.go

### PR DESCRIPTION
**Pull request by dongzhiwei-git**

### CODE REVIEW

File: db_test.go [hunk 1]
```diff
@@ -207,11 +207,12 @@ func TestDB_Merge_For_string(t *testing.T) {
 	db2, err := Open(
 		DefaultOptions,
 		WithDir(fileDir),
+		WithNodeNum(1011),
 		WithSegmentSize(1*100),
 	)
 
 	require.NoError(t, err)
-
```
Comment (elliotchenzichang) [inline db_test.go:214]: Seems this function is for testing the merge function, not for testing the Options function. Could you help add it in the rightful place?

**Review by elliotchenzichang — COMMENTED** [review]
Left a comment.

File: db_test.go (hunk 1 above)
Comment (dongzhiwei-git) [reply db_test.go:214]: ok

Comment (dongzhiwei-git) [reply db_test.go:214]: Resolved

**Review by elliotchenzichang — COMMENTED** [review]
LGTM

**Comment by elliotchenzichang** [comment]
LGTM, thanks for your contribution.

**Pull request merged**